ts/src/test/test.ts

```diff
@@ -235,15 +240,15 @@ export default class testMainClass extends baseMainTestClass {
 
     async testMethod (methodName, exchange, args, isPublic) {
         const methodNameInTest = getTestName (methodName);
-        // if this is a private test, and the implementation was already tested in public, then no need to re-test it in private test (exception is fetchCurrencies, because our approach in exchange)
+        // if this is a private test, and the implementation was already tested in public, then no need to re-test it in private test (exception is fetchCurrencies, because our approach in base exchange)
         if (!isPublic && (methodNameInTest in this.checkedPublicTests) && (methodName !== 'fetchCurrencies')) {
             return;
         }
         let skipMessage = undefined;
         const isFetchOhlcvEmulated = (methodName === 'fetchOHLCV' && exchange.has['fetchOHLCV'] === 'emulated'); // todo: remove emulation from base
         if ((methodName !== 'loadMarkets') && (!(methodName in exchange.has) || !exchange.has[methodName]) || isFetchOhlcvEmulated) {
             skipMessage = '[INFO:UNSUPPORTED_TEST]'; // keep it aligned with the longest message
-        } else if (methodName in this.skippedMethods) {
+        } else if ((methodName in this.skippedMethods) && (typeof this.skippedMethods[methodName] === 'string')) {
```

Comment:
@ttodua why do we need to use typeof here?

Reply:
because we added granular skip support, so now we can fully skip method:
`fetckTicker: "the endpoint is buggy"`
or as object:
```fetchTickers: {
'high': "skip this field, because exchange does not provide this value correctly..."
}```

Reply:
sorry, I think I don't understand what you mean here?

Reply:
if you mean `if (this.skippedMethods[methodName])` we dont use this alone anymore - we use it in conjunction with "type of" string, to skip whole method or not. if `type` is not string, we assume it to be an object.

Reply:
so, that's why inside tests (251th line you have highlighted) is only meant to skip "whole method" or not. that line either skips method fully, or allows it. whenever it's allowed, then we pass skipProps  ([can be empty `{}` object too](https://github.com/ccxt/ccxt/blob/1248361c6f11bb88fc4bf8147790bb60c8387298/ts/src/test/test.ts#L268))  to test method

Reply:
> line

@ttodua oh ok I got it now, but yeah this might be a bit confusing 😅